modules/auxiliary/scanner/rdp/cve_2019_0708.rb

```diff
@@ -0,0 +1,545 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'digest'
+require 'rc4'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'CVE-2019-0708 Microsoft Remote Desktop RCE Checker',
+      'Description'    => %q{
+        This module checks a range of hosts for the CVE-2019-0708 vulnerability.
+        This should not cause a DoS on the target.
+      },
+      'References'     =>
+        [
+          [ 'CVE', '2019-0708' ],
+          [ 'URL', 'https://portal.msrc.microsoft.com/en-US/security-guidance/advisory/CVE-2019-0708' ]
```

Comment:
Probably this may help too.
[ 'URL', 'https://wazehell.io/2019/05/22/cve-2019-0708-technical-analysis-rdp-rce/' ]